package.json

```diff
@@ -13,6 +13,7 @@
     "@polkadot/util-crypto": "^8.7.1",
     "chalk": "^4.1.2",
     "esbuild": "^0.13.8",
+    "lodash.chunk": "^4.2.0",
```

Comment:
I think loading lodash just for chunk seems a bit too much no ?

Reply:
This is only importing `lodash.chunk`, so not the entire library. Should be lite imo.